libs/datamodel/core/tests/attributes/default_positive.rs

```diff
@@ -10,7 +10,7 @@ fn should_set_default_for_all_scalar_types() {
     model Model {
         id Int @id
         int Int @default(3)
-        float Float @default(3.14)
+        float Float @default(3.20)
```

Comment:
? 😁

Reply:
Ah, this was flagged as "this looks too closely like PI, use the PI constant instead" -- given it's just a random value, changing it seemed easier.